app/src/main/res/layout/scene_no_record.xml

```diff
@@ -19,26 +19,54 @@
         <androidx.constraintlayout.widget.ConstraintLayout
             android:layout_width="match_parent"
             android:layout_height="wrap_content">
+            <View
+                android:id="@+id/viewRectangleLine"
```

Comment:
we can give id name in small letters with underscore like this: view_rectangle_line so same naming convention can be followed in whole project.